content-src/components/DiscoveryStreamComponents/TopSites/_TopSites.scss

```diff
@@ -11,6 +11,18 @@
 
       .top-site-inner > a:-moz-any(.active, :focus) .tile {
         @include ds-fade-in;
+
+        @include dark-theme-only {
+          @include ds-fade-in($blue-40-40);
+        }
+      }
+
+      .top-site-inner > a:-moz-any(:hover) .tile {
+        @include ds-fade-in($grey-30);
```

Comment:
With this there is slight flicker  because of border-radius changing from 6px to 4px on hover. Changing variable $top-sites-border-radius to 4px should fix it